frontend/app/controllers/sign-in.js

```diff
@@ -14,7 +14,7 @@ export default Ember.Controller.extend({
     this.reset();
     if (response.status==true){
       this.reset();
-      window.location.href = "/userHome";
+      window.location.href = "/user-home";
```

Comment:
Instead of this, you should use `this.transitionToRoute('user-home')`

Documentation: https://www.emberjs.com/api/ember/2.14/classes/Ember.Controller/methods/transitionToRoute?anchor=transitionToRoute